airbyte-webapp/src/components/StreamTestingPanel/LogsDisplay.tsx

```diff
@@ -0,0 +1,29 @@
+import { FormattedMessage } from "react-intl";
+
+import { Text } from "components/ui/Text";
+
+import { StreamReadLogsItem } from "core/request/ConnectorBuilderClient";
+
+import styles from "./LogsDisplay.module.scss";
+
+interface LogsDisplayProps {
+  logs: StreamReadLogsItem[];
+}
+
+export const LogsDisplay: React.FC<LogsDisplayProps> = ({ logs }) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.header}>
+        <Text size="sm" bold>
+          <FormattedMessage id="connectorBuilder.connectorLogs" />
+        </Text>
+        <Text className={styles.numLogsDisplay} size="xs" bold>
+          {logs.length}
+        </Text>
+      </div>
+      <div className={styles.logsDisplay}>
+        <pre>{JSON.stringify(logs, null, 2)}</pre>
```

Comment:
Minor nitpick (not necessarily needs to be solved in this PR): `JSON.stringify` (and `JSON.parse`) are rather expensive operations. If `logs` tend to be long, we would want to make sure to memoize the `JSON.stringify`, since it could get expensive otherwise.